### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by DuttaAnik** [steineggerlab/Metabuli#95]
Hi,
Thanks a lot for developing such a cool tool with a standalone app. I saw that after the classification, one can Extract species-specific fastq reads from the app. Would it be possible to include a function to convert the fastq into a fasta file? Then one can easily use that fasta file for a blast search.

On another note, the NCBI and assembly link on the top right corner of the App does not redirect to the correct organism on NCBI.

**Comment by jaebeom-kim** [comment]
Thank you for giving a good idea!
I think we can make a check box to select between FASTQ/FASTA when extracting reads!

>On another note, the NCBI and assembly link on the top right corner of the App does not redirect to the correct organism on NCBI.

I think it can happen when the GTDB is used for search.
Please let me know if you met the error with NCBI-based databases.
We will soon make a proper link for GTDB !

Thank you so much again!

**Comment by DuttaAnik** [comment]
Thank you for accepting the request. It will help the analysis smoother. 
Yes, I used the GTDB that's why I did not get the correct link. I have not tried with NCBI database though.
However, there is one more thing. I saw that one can only upload one report file at a time to visualize the results. In Pavian, it is possible to upload multiple report files. 
Are you also planning to expand the current functionality to allow uploading multiple report files?
Thank you.

**Comment by jaebeom-kim** [comment]
Yes, we are planning to support uploading multiple report files.

For selecting between FASTQ/FASTA when extracting reads, `--extract-format` parameter is added in the command line Metabuli, and it will be added to Metabuli App soon.
